components/Tabs/Tabs.js

```diff
@@ -1,62 +1,53 @@
-
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element = element;
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = element.dataset.tab;
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+    this.itemElement = document.querySelector(
+      `.tabs-item[data-tab="${this.data}"]`
+    );
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
+    this.tabItem = new TabItem(this.itemElement);
     // Add a click event listener on this instance, calling the select method on click
-
-  };
+    element.addEventListener("click", () => this.select());
+  }
 
   select() {
     // Get all of the elements with the tabs-link class
-    // const links;
-
+    const links = [...document.querySelectorAll(".tabs-link")]; // const links = ...
```

Comment:
ooh, very nice use of the spread syntax here! does the job job that iterating over the nodeList with `map` would but a little more succinct. well done!